javascript/javascript1/week1/age_ify.js

```diff
@@ -0,0 +1,21 @@
+// a Future age calculator
+const yearOfBirth=1995;
+const yearOfFuture=2027;
+const age=yearOfFuture-yearOfBirth;
+console.log("You will be "+age+" years old in "+yearOfFuture);  
+
+
+
+
+//a dog age calculator
+const dogYearOfBirth=2025;
+const dogYearOfFuture=2045;
+const dogAge=dogYearOfFuture-dogYearOfBirth;
+let shouldShowResultInDogYears=true;
```

Comment:
Use `const` instead of `let`  shouldShowResultInDogYears is not reassigned, so `const` is preferred.